common/djangoapps/edeos/api_calls.py

```diff
@@ -140,9 +145,13 @@ def post(self, url, payload, headers=None, can_retry=True):
         if headers is not None:
             headers_.update(headers)
         resp = requests.post(url, json=payload, headers=headers_)
-        log.info("Edeos response: status {}, content {}".format(resp.status_code, resp.content))
+        log.info("Edeos response: status {}, content {}".format(resp.status_code, resp.content)) #TODO this response will be in the termilall and it is not convenient maybe not in the teminal? 
+        
         if resp.status_code in (httplib.OK, httplib.CREATED):
-            return resp.json()
+            try:
+                return resp.json()
+            except JSONDecodeError:
```

Comment:
Just a note, for future references: couldn't avoid this try-except here, as profile_statistics is very peculiar (for now, it's a single endpoint that doesn't return JSON )